kokoro/ubuntu/periodic.sh

```diff
@@ -17,3 +17,6 @@ cd dashboard
 # For all versions available in Maven Central and local repository
 mvn -V -B exec:java -Dexec.mainClass="com.google.cloud.tools.opensource.dashboard.DashboardMain" \
   -Dexec.arguments="-a com.google.cloud:libraries-bom"
+
+mvn -V -B exec:java -Dexec.mainClass="com.google.cloud.tools.opensource.dashboard.DashboardMain" \
+  -Dexec.arguments="-a com.google.cloud:gcp-lts-bom"
```

Comment:
Memo for myself: https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/kokoro/ubuntu/periodic.cfg references this shell script. The `google3/devtools/kokoro/config/prod/cloud-opensource-java/ubuntu/common.cfg;l=16` references the cfg file.